packages/uswds-core/src/styles/settings/_settings-color.scss

```diff
@@ -119,6 +119,9 @@ $theme-color-disabled-family: "gray" !default;
 $theme-color-disabled-light: "gray-10" !default;
 $theme-color-disabled: "gray-20" !default;
 $theme-color-disabled-dark: "gray-30" !default;
```

Comment:
Note: `disabled-light` and `disabled-dark` have ambiguous use cases and aren't currently referenced in USWDS. I chose to go ahead and create new settings variables rather than repurposing the existing vars just to minimize potential user impact, but I can see that a case could be made to the contrary. Flagging in case anyone has thoughts.